src/lib/mongoBackend/mongoRegisterContext.cpp

```diff
@@ -109,9 +109,10 @@ HttpStatusCode mongoRegisterContext
     return SccOk;
   }
 
-  const char* colName = getRegistrationsCollectionName(tenant).c_str();
+  const std::string    colName  = getRegistrationsCollectionName(tenant);
+  const mongo::BSONObj bson     = BSON("_id" << id << REG_SERVICE_PATH << sPath);
 
-  if (!collectionFindOne(colName, BSON("_id" << id << REG_SERVICE_PATH << sPath), &reg, &err))
+  if (!collectionFindOne(colName, bson, &reg, &err))
```

Comment:
Fixed in e294564 (and the two following commits that fixed mistakes: 8923a76 and a9ed96b)